cob_hardware_config/cob4-3/config/base_controller.yaml

```diff
@@ -1,5 +1,5 @@
 ## joint_names
-joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, fr_caster_rotation_joint, fr_caster_r_wheel_joint, b_caster_rotation_joint, b_caster_r_wheel_joint]
+joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, b_caster_rotation_joint, b_caster_r_wheel_joint, fr_caster_rotation_joint, fr_caster_r_wheel_joint]
```

Comment:
I don't mean merge conflict....I mean the fact, that in #447 @ipa-nhg pushed the version that was used when shipping the platforms